### DISCUSSION TITLE
Combining commands in vimium://run

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [gdh1995/vimium-c#1058] [Q&A] [answered]
How do I do the following:
1. Switch tab (Say I wanna go to third tab)
2. Open a URL with query

I was thinking of something along the lines `vimium://run/3g0+openUrl url=<url>`. Of course, this doesn't work.

**Comment by limaceous-bushwhacker** [comment]
Looks like this is the proper syntax: `vimium://run/2g0+openUrl#url="<URL>=$s"&reuse="current"`. Is there a way to check if $s is empty or not and only then open the URL?

**Reply by gdh1995** [reply, marked as answer]
1. Yes it's `#` to seperate command name and its parameters in `vimium://run`.
2. Sorry I didn't get what you want by `check $s`.

If you want to:
- if query is not empty, then execute the composed command filled with query words
- if query is empty, then do nothing

Then you may configure such a "custom search engine":
```
# “blank” is a special command meaning "to do nothing"
g: vimium://run/2g0+openUrl#reuse=current&#url=<URL-with-$s> blank=vimium://run/blank
```
